packages/block-library/src/image/editor.scss

```diff
@@ -63,10 +63,11 @@ figure.wp-block-image:not(.wp-block) {
 		transform: translate(-50%, -50%);
 	}
 
-	// When the Image block is linked,
-	// it's wrapped with a disabled <a /> tag.
-	// Restore cursor style so it doesn't appear 'clickable'.
 	> a {
+		pointer-events: none;
+		// When the Image block is linked,
+		// it's wrapped with a disabled <a /> tag.
+		// Restore cursor style so it doesn't appear 'clickable'.
 		cursor: default;
```

Comment:
```suggestion
		// When the Image block is linked,
		// it's wrapped with a disabled <a /> tag.
		// Restore cursor style so it doesn't appear 'clickable'
		// and remove pointer events. Safari needs the display property.
		pointer-events: none;
		cursor: default;
		display: inline-block;
```

Reply:
Maybe we could group the styles so folks know that they belong to the same "change".

What do you think?

Also pointer events none doesn't work in Safari for me with the display property.